test/unittests/execute_test.cpp

```diff
@@ -117,6 +117,42 @@ TEST(execute, local_tee)
     EXPECT_EQ(ret[0], 42);
 }
 
+TEST(execute, global_get)
+{
+    fizzy::Module module;
+    fizzy::Global g{true, fizzy::GlobalInitType::constant, {42}};
+    module.globalsec.emplace_back(g);
```

Comment:
Can you squash these two?

Reply:
It won't compile, because it can't deduce the type of `{42}`

Reply:
```
module.globalsec.emplace_back(fizzy::Global{true, fizzy::GlobalInitType::constant, {42}});
``

won't compile ?

Reply:
this one will, thanks